tigramite/independence_tests/independence_tests_base.py

```diff
@@ -701,9 +698,15 @@ def _get_p_value(self, val, array, xyz, T, dim,
             pval = self.get_analytic_significance(value=val, T=T, dim=dim, xyz=xyz)
         # Check if we are using the shuffle significance
         elif use_sig == 'shuffle_test':
-            pval = self.get_shuffle_significance(array=array,
-                                                 xyz=xyz,
-                                                 value=val)
+            if data_type is None:
+                pval = self.get_shuffle_significance(array=array,
+                                                     xyz=xyz,
+                                                     value=val)
+            else:
+                pval = self.get_shuffle_significance(array=array,
+                                                     xyz=xyz,
+                                                     value=val,
+                                                     data_type=data_type)
```

Comment:
Doesn't get_shuffle_significance have data_type=None as an argument and so we don't need the if-statement, but just add data_type=data_type?